crates/ruff_linter/src/rules/airflow/helpers.rs

```diff
@@ -0,0 +1,64 @@
+use crate::rules::numpy::helpers::ImportSearcher;
+use ruff_python_ast::statement_visitor::StatementVisitor;
+use ruff_python_ast::{Expr, ExprName, StmtTry};
+use ruff_python_semantic::Exceptions;
+use ruff_python_semantic::SemanticModel;
+
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub(crate) enum Replacement {
+    None,
+    Name(&'static str),
+    Message(&'static str),
+    AutoImport {
+        path: &'static str,
+        name: &'static str,
+    },
+}
+
+pub(crate) fn is_guarded_by_try_except(
+    expr: &Expr,
+    replacement: &Replacement,
+    semantic: &SemanticModel,
+) -> bool {
+    match expr {
+        Expr::Name(ExprName { id, .. }) => {
```

Comment:
Is this always going to be a `Name` node? Can it be an `Attribute` node?

For example, in NumPy deprecation rules, there's this test case:

https://github.com/astral-sh/ruff/blob/ae2cf91a360e927f5da307d0290be1a6229ced9a/crates/ruff_linter/resources/test/fixtures/numpy/NPY201_2.py#L71-L74

Is something like that possible? It's totally fine if you don't want to support something like that.

Reply:
We'll want to add it. But I'm thinking of addressing it in another PR. @sunank200 will  take over during the time I'm out.

Reply:
Sounds good. Should I go merge this PR then?

Enjoy your time off!

Reply:
Yep, it would be super helpful if we could merge it first (also unblock a few draft PR). Let me resolve the conflict now!

Reply:
Updated 🙌